src/info_command.c

```diff
@@ -157,6 +157,29 @@ int IndexInfoCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
       geom_idx_sz += api->report(idx);
     }
 
+    if (FIELD_IS(fs, INDEXFLD_T_VECTOR)) {
+      VecSimParams vec_params = fs->vectorOpts.vecSimParams;
+      VecSimAlgo algo = vec_params.algo;
+
+      if (algo == VecSimAlgo_TIERED) {
+        VecSimParams *backend_params = vec_params.algoParams.tieredParams.primaryIndexParams;
```

Comment:
You can save some code if you use:
```
if (algo == VecSimAlgo_TIERED) {
        params = *vec_params.algoParams.tieredParams.primaryIndexParams;
}
```
then reply for the algorithm, and then continue with the rest as you did (separate by algo)